Readme.md

```diff
@@ -1,95 +1,97 @@
-LibSass
-=======
+LibSass - Sass compiler written in C++
+======================================
 
 by Aaron Leung ([@akhleung]), Hampton Catlin ([@hcatlin]), Marcel Greter ([@mgreter]) and Michael Mifsud ([@xzyfer])
```

Comment:
It's worth distinguishing current maintainers. People still contact Hampton directly who then proxies messages on to me.